packages/web-ui/src/providers/ProjectProvider.tsx

```diff
@@ -13,11 +15,14 @@ const ProjectContext = createContext<IProjectContextType>(
 const ProjectProvider = ({
   children,
   project,
+  latitudeApiKey,
 }: {
   children: ReactNode
+  latitudeApiKey: string
 } & IProjectContextType) => {
+  const sdkRef = useRef(new LatitudeSdk({ latitudeApiKey }))
```

Comment:
⚠️ This is not the right approach. The SDK has to be used on the server. So I'll move it to a server action to be used in the playground. 

Is not good idea to use in client because we expose the api key. Inside playground is less important but still a security concern.